packages/docusaurus/src/commands/deploy.ts

```diff
@@ -158,21 +159,28 @@ export async function deploy(
 
         const commitMessage =
           process.env.CUSTOM_COMMIT_MESSAGE ||
-          `Deploy website version based on ${currentCommit}`;
+          `Deploy website - based on ${currentCommit}`;
         const commitResults = shell.exec(`git commit -m "${commitMessage}"`);
         if (
           shell.exec(`git push --force origin ${deploymentBranch}`).code !== 0
         ) {
           throw new Error('Error: Git push failed');
         } else if (commitResults.code === 0) {
           // The commit might return a non-zero value when site is up to date.
-          const websiteURL =
-            githubHost === 'github.com'
-              ? // gh-pages hosted repo
-                `https://${organizationName}.github.io/${projectName}`
-              : // GitHub enterprise hosting.
-                `https://${githubHost}/pages/${organizationName}/${projectName}`;
-          shell.echo(`Website is live at: ${websiteURL}`);
+          let websiteURL = '';
```

Comment:
I'm going to revert this section since it's not a trivial change. You can submit a new PR just for these changes.